notifier/pull.py

```diff
@@ -27,11 +27,11 @@ def _http_post(*a, **kw):
     try:
         logger.debug('POST %s %s', a[0], kw)
         response = requests.post(*a, **kw)
-    except requests.exceptions.ConnectionError, e:
+    except requests.exceptions.ConnectionError as e:
         _, msg, tb = sys.exc_info()
-        raise CommentsServiceException, "comments service request failed: {}".format(msg), tb
+        raise CommentsServiceException("comments service request failed: {}".format(msg), tb)
```

Comment:
This isn't quite the right update: the traceback shouldn't be a second argument to the constructor.  We should skip this line for now, and just leave it as it was.

This repo is small enough that we will just convert it to being Python-3-only when the time comes.

Reply:
Got it...  I think you are looking for [__six.reraise()__](https://six.readthedocs.io/index.html?highlight=reraise#six.reraise) here.